### DISCUSSION TITLE
DateTime Labeler Exception

### DISCUSSION THREAD

**Discussion by En-JON-eer** [Live-Charts/LiveCharts2#916] [Q&A]
![image](https://user-images.githubusercontent.com/89985196/216050963-8d2b76ca-6008-4ea1-b866-5e354d71cdb4.png)

I am seeing this exception pop up a lot. But I'm not really sure where to start with it...

**Comment by torum** [comment]
I was getting the same error a lot from [my app](https://github.com/torum/BitWallpaper). But I manage to avoid it somehow. It seemed exception occurs when the "value" is null. I think the trick was assigning dummy initial value.
